contracts/addresses/addresses.go

```diff
@@ -5,7 +5,7 @@ import "github.com/ethereum/go-ethereum/common"
 var (
 	CeloTokenAddress            = common.HexToAddress("0x471ece3750da237f93b8e339c536989b8978a438")
 	FeeHandlerAddress           = common.HexToAddress("0xcd437749e43a154c07f3553504c68fbfd56b8778")
-	FeeCurrencyDirectoryAddress = common.HexToAddress("0x71ffbd48e34bdd5a87c3c683e866dc63b8b2a685")
+	FeeCurrencyDirectoryAddress = common.HexToAddress("0x9212Fb72ae65367A7c887eC4Ad9bE310BAC611BF")
```

Comment:
Why are we updating the address?

Reply:
Primitives redeployed the contract, not sure why exactly.